docs/frontend.md

```diff
@@ -120,6 +120,33 @@ to deduplicate resolved package versions within the Yarn lockfile.
 
 See [`@18f/identity-i18n` package documentation](../app/javascript/packages/i18n/README.md).
 
+### Analytics
+
+See [`@18f/identity-analytics` package documentation][analytics_package] for code examples detailing
+how to track an event in JavaScript.
+
+Any event logged from the frontend must be added to the `EVENT_MAP` allowlist in [`FrontendLogController`][frontend_log_controller.rb].
```

Comment:
I feel like the **must me added to `EVENT_MAP`** bit is not quite right, because we still have the fallback to the `Frontend: ` prefix

https://github.com/18F/identity-idp/blob/9c17164c0b8d9b4aefad74dde1a521c111b53aac/app/services/frontend_logger.rb#L25-L26


Maybe something like...

```suggestion
Events event logged from the frontend should be added to the `EVENT_MAP` allowlist in [`FrontendLogController`][frontend_log_controller.rb] so they can be documented in `AnalyticsEvents`, otherwise 
they will be prefixed with `"Frontend: #{name}"`
```

Reply:
You’re right about that. I should have mentioned in the opening comment, but I wrote this aspirationally toward wanting to remove that behavior. I’m away from my computer but there was a related discussion earlier in the appdev channel where @mitchellhenke is working on a branch to remove it.

Reply:
Related Slack discussion: https://gsa-tts.slack.com/archives/C0NGESUN5/p1697664301580629